AGENTS.md

```
@@ -0,0 +1,203 @@
+**Purpose**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)